Sources/Xcodeproj/pbxproj().swift

```diff
@@ -55,7 +55,7 @@ func xcodeProject(
     // Determine the source root directory (which is NOT necessarily related in
     // any way to `xcodeprojPath`, i.e. we cannot assume that the Xcode project
     // will be generated into to the source root directory).
-    let sourceRootDir = graph.rootPackage.path
+    let sourceRootDir = graph.rootPackages[0].path
```

Comment:
I agree, right now the invariant is that all graph will have exactly one root package.